Modules/Scripted/QReads/QReads.py

```diff
@@ -385,10 +399,10 @@ class QReadsLogic(ScriptedLoadableModuleLogic):
   }
 
   WINDOW_LEVEL_PRESETS = {
-    'CT-BodySoftTissue': (1600, -600),
+    'CT-BodySoftTissue': (400, 40),
     'CT-Bone': (2500, 300),
     'CT-Head': (100, 40),
-    'CT-Lung': (400, 40)
+    'CT-Lung': (1600, -600)
```

Comment:
These changes have already been integrated in https://github.com/KitwareMedical/SlicerQReads/commit/40d16a7384b0653256dc8ca9b13fc15e2679eabe